gstd/gstd_element.c

```diff
@@ -381,7 +382,12 @@ gstd_element_property_get_type (GType g_type)
   //add details. For example, int properties can display their max and min
   //values, flags and enums could display the options, etc... Similar to
   //what gst-inspect does
-  return GSTD_TYPE_PROPERTY;
+
+  if (g_type == G_TYPE_ARRAY) {
```

Comment:
it’s better if we use the switch below

Reply:
The switch can't handle G_TYPE_ARRAY because it doesn't return an int value. That's why I add the conditional.

Reply:
The switch can't handle G_TYPE_ARRAY because it doesn't return an int value. That's why I add the conditional.